platform/platform-filter-delegate/src/test/java/org/codice/ddf/platform/filter/delegate/DelegateServletFilterTest.java

```diff
@@ -329,6 +327,11 @@ public void testRemoveSecurityFilter() throws IOException, ServletException {
     }
 
     @Test
+    public void testRemoveNullSecurityFilter() throws IOException, ServletException {
+        delegateServletFilter.removeSecurityFilter(null);
```

Comment:
In Junit tests, I usually just put a line comment at the end where my assertion would go had I been writing a Spock test.